src/components/Tooltip.js

```diff
@@ -1,97 +1,111 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Transition } from "react-transition-group";
 import styled from "styled-components";
 
 import { PORTAL_TARGETS } from "constants/domNodes";
 import { PALETTE, Z_INDEXES } from "constants/styles";
 
 import { renderToPortal } from "helpers/renderToPortal";
 
+// For the arrow, it should be 4px (.25rem) away from the edge. Since it's a box
+// rotated 45 degrees, calculate the length of 1 side of the box.
+// a^2 + b^2 = c^2
+const hypotenuse = (0.25 ** 2 + 0.25 ** 2) ** 0.5;
```

Comment:
Mrs. Caulfield, my high school geometry teacher, salutes you

Reply:
I DEFINITELY did not try to use SOHCAHTOA first